eng/Versions.props

```diff
@@ -123,8 +123,9 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/dotnet/templating -->
-    <MicrosoftTemplateEngineCliPackageVersion>7.0.100-rc.1.22379.2</MicrosoftTemplateEngineCliPackageVersion>
     <MicrosoftTemplateEngineAbstractionsPackageVersion>7.0.100-rc.1.22379.2</MicrosoftTemplateEngineAbstractionsPackageVersion>
+    <MicrosoftTemplateEngineEdgePackageVersion>7.0.100-rc.1.22379.2</MicrosoftTemplateEngineEdgePackageVersion>
```

Comment:
This change requires dependencies to be changed as well, otherwise it won't be updated automatically. Since all the packages are updated at the same time I suggest we stick to single dependency.

```
  <PropertyGroup>
    <!-- Dependencies from https://github.com/dotnet/templating -->
    <MicrosoftTemplateEngineAbstractionsPackageVersion>7.0.100-rc.1.22401.3</MicrosoftTemplateEngineAbstractionsPackageVersion>
    <MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>$(MicrosoftTemplateEngineAbstractionsPackageVersion)</MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion>
    <MicrosoftTemplateEngineUtilsPackageVersion>$(MicrosoftTemplateEngineAbstractionsPackageVersion)</MicrosoftTemplateEngineUtilsPackageVersion>
and so on
  </PropertyGroup>

```